Gemfile.lock

```diff
@@ -394,7 +394,7 @@ DEPENDENCIES
   coffee-rails
   dalli
   database_cleaner
-  devise
+  devise (= 3.5.6)
```

Comment:
Well, we'll need another consensus then. I've been bitten by soft dependencies so much that I don't understand why it's not the Gemfile default in official examples. You must ensure which version you're using and yes, I don't trust maintainers.